src/foam/dao/journal/CommentJournalEntry.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
I don't think we need this since what we used to store in comments is now stored in the JournalEntry itself.

Reply:
We should think about whether we want the comment in the journal entry or in the object. We recently moved it into the object with the LastModifiedByAware stuff but I'm not sure if that's the best place for it because now it requires us to add these traits to each of those objects. If this info is just for audit purposes, I think we should do it this way because it'll apply to everything that's put into a dao and the object doesn't need to care when it was last modified. It makes for slightly less developer overhead because we won't need to worry about adding the various implements on each object.

Reply:
This seems reasonable for two reasons:
1. retains current auditing information without immediately requiring us to update all objects with Created/LastModified.
2. This along with the UserComment decorator can be disable where it's not needed/wanted.